packages/headless/src/controllers/product-listing/facet/headless-product-listing-facet-manager.ts

```diff
@@ -5,12 +5,15 @@ import {buildController} from '../../controller/headless-controller';
 import {
   FacetManager,
   FacetManagerPayload,
+  FacetManagerState,
 } from '../../facet-manager/headless-facet-manager';
 import {ProductListingSection} from '../../../state/state-sections';
 import {ProductListingEngine} from '../../../app/product-listing-engine/product-listing-engine';
 
+export {FacetManager, FacetManagerState, FacetManagerPayload};
+
 /**
- * Creates a `FacetManager` instance.
+ * Creates a `FacetManager` instance for the product listing.
```

Comment:
Could the `for the product listing` phrase could be removed, since it would be implied by the import path? e.g.

```
import {buildFacetManager} from '@coveo/headless/product-listing';
```

Reply:
I saw some value in putting it there because it uses the same name as the original build controller methods.

So when you check the comment, you would see `for the product listing` for this one, but not in the standard headless one.

But I'm not strongly attached to it, I can removed them all

Reply:
Ok, I leave the decision to the curator of our documentation, @dlutzCoveo.